### ISSUE TITLE
False State color

### ISSUE THREAD

**Issue by aquintiliano** [yuyang041060120/angular2-ui-switch#14]
First of all, congrats, it's a great and easy-to-use component.

You let we change the True state color, but I want to change the false state color as well

**Comment by misadjalovic** [comment]
Yeah I tried as well to change the  false state bg-color but no luck even with the change of the bg-color in the component css for the span element.

**Comment by biskis** [comment]
You can change the false (or off) color using `switchOffColor` param (no need to do anything in CSS)

For example, this will be green when on and red when off with white background. 

`<ui-switch color="#fff" switchColor="green" switchOffColor="red"></ui-switch>`

**Comment by fabrizziocht** [comment]
but also we need to change defaultBgColor, please update this value with @Input decorator
